hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/io/KeyOutputStream.java

```diff
@@ -457,6 +460,9 @@ public void hsync() throws IOException {
       throw new UnsupportedOperationException("The replication factor = "
           + replication.getRequiredNodes() + " <= 1");
     }
+    if (ozoneManagerVersion.compareTo(OzoneManagerVersion.HBASE_SUPPORT) < 0) {
+      throw new IOException("OzoneManager does not support hsync");
```

Comment:
I feel the error message in below ListOpenFilesSubCommand.java carries more info. 

"It requires OzoneManager version " +  OzoneManagerVersion.HBASE_SUPPORT.name().